tests/test_marquez_client.py

```diff
@@ -258,6 +258,75 @@ def test_create_job(self, mock_put):
         assert str(response['id']) is not None
         assert str(response['location']) == location
 
+    @mock.patch("marquez_client.client.MarquezClient._put")
+    def test_create_job_with_run_id(self, mock_put):
+        run_id = str(uuid.uuid4())
+        job_name = "my-job"
+        input_dataset = [
+            {
+                "namespace": "my-namespace",
+                "name": "public.mytable"
+            }
+        ]
+        output_dataset = {
+            "namespace": "my-namespace",
+            "name": "public.mytable"
+        }
+
+        location = "https://github.com/my-jobs/blob/" \
+                   "07f3d2dfc8186cadae9146719e70294a4c7a8ee8"
+
+        context = {
+            "sql": "SELECT * FROM public.mytable;"
+        }
+
+        mock_put.return_value = {
+            "id": {
+                "namespace": "my-namespace",
+                "name": "my-job"
+            },
+            "type": "BATCH",
+            "name": "my-job",
+            "createdAt": "2020-08-12T07:30:55.321059Z",
+            "updatedAt": "2020-08-12T07:30:55.333230Z",
+            "namespace": "my-namespace",
+            "inputs": [
+                {
+                    "namespace": "my-namespace",
+                    "name": "public.mytable"
+                }
+            ],
+            "outputs": [
+                {
+                    "namespace": "my-namespace",
+                    "name": "public.mytable"
+                }
+            ],
+            "location": "https://github.com/my-jobs/blob/"
+                        "07f3d2dfc8186cadae9146719e70294a4c7a8ee8",
+            "context": {
+                "sql": "SELECT * FROM public.mytable;"
+            },
+            "description": "My first job.",
+            "latestRun": None,
+            "runId": run_id
+        }
+
+        response = self.client.create_job(
+            namespace_name=_NAMESPACE,
+            job_name=job_name,
+            job_type=JobType.BATCH,
+            location=location,
+            input_dataset=input_dataset,
+            output_dataset=output_dataset,
+            context=context,
+            run_id=run_id
+        )
+
+        assert str(response['id']) is not None
+        assert str(response['location']) == location
+        assert str(response['runId']) == run_id
+
```

Comment:
Mind if we also add an assert that the `mock_put` was called only once?

Reply:
it's redundant. but if you insist. why not, sure.

Reply:
done.